### DISCUSSION TITLE
风控怎么解决？

### DISCUSSION THREAD

**Discussion by daizihan233** [Mrs4s/go-cqhttp#1745] [Q&A]
之前挂了七天无果，机器人一直都没发过啥违规的东西，10月初的时候突然被风控（或许因为二十大？）
换了好几个号，基本全被风控，删token改device均无果

**Comment by Synex93** [comment]
一样

**Comment by daizihan233** [comment]
补：2022/10/30 最新版 dev 仍然风控
